chain/chain-primitives/src/error.rs

```diff
@@ -146,6 +146,9 @@ pub enum Error {
     /// `next_bps_hash` doens't correspond to the actual next block producers set
     #[error("Invalid Next BP Hash")]
     InvalidNextBPHash,
+    /// The block has a protocol version that's outdated
+    #[error("Invalid protocol version")]
+    InvalidProtocolVersion,
```

Comment:
nit: everywhere else we call it 'outdated' not 'invalid'

Reply:
Yeah but all the other errors starts with InvalidXXX... I think I'll stick with Invalid